tools/src/format.rs

```diff
@@ -291,8 +291,10 @@ pub fn format_file_data(cfg: &config::Config,
 
     if let Some(ext) = path_wrapper.extension() {
         if ext.to_str().unwrap() == "svg" {
-            let url = format!("{}/raw-file/tip/{}", config::get_hg_root(tree_config), path);
-            try!(output::generate_svg_preview(writer, &url))
+            if let Some(ref hg_root) = tree_config.paths.hg_root {
+                let url = format!("{}/raw-file/tip/{}", hg_root, path);
+                try!(output::generate_svg_preview(writer, &url))
```

Comment:
If while you're here you want to untry it (using `?` instead) that'd be great :)

Reply:
Done